addon/templates/components/polaris-data-table.hbs

```diff
@@ -28,7 +35,10 @@
 
         {{#if totals}}
           <tr>
-            {{#each (polaris-data-table/insert-presentational-cell totals) as |total index|}}
+            {{#each
+              (polaris-data-table/insert-presentational-cell totals)
+            as |total index|
```

Comment:
doesn't this line look off?

Reply:
I agree....but this was erroring the linter....checked again now, it doesn't...weird
will fix